src/components/Sidebar.js

```diff
@@ -20,7 +23,7 @@ const ThoughtsTab = ({ thoughtsRanked }) => {
 }
 
 const RecentEdited = () => {
-  const recentlyEdited = _.sortedUniqBy(_.reverse(_.sortBy(useSelector(state => (state.recentlyEdited)), 'lastUpdated')), recentThought => hashContext(recentThought.path)) // eslint-disable-line fp/no-mutating-methods
+  const recentlyEdited = _.sortedUniqBy(_.reverse(_.sortBy(findTreeDescendants(useSelector(state => (state.recentlyEdited)), [EM_TOKEN]), 'lastUpdated')), recentThought => hashContext(recentThought.path)) // eslint-disable-line fp/no-mutating-methods
```

Comment:
We can remove `_.sortedUniqBy` since the tree eliminates duplicates now.

Should also be able to get rid of the `eslint-disable-line` as well.

Reply:
Sure.